From 1661ec2fd8296c899fea34eeba07efeda635cc79 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Wed, 21 Nov 2018 19:13:08 +0000
Subject: [PATCH] MANTA-4024 madtom doesn't use its nodeunit dependency (fix
 build) Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 9be8549..0ca1640 100644
--- a/Makefile
+++ b/Makefile
@@ -73,7 +73,7 @@ NPM_ENV		 = MAKE_OVERRIDES="CTFCONVERT=/bin/true CTFMERGE=/bin/true"
 # Repo-specific targets
 #
 .PHONY: all
-all: $(SMF_MANIFESTS) | $(REPO_DEPS) scripts
+all: $(SMF_MANIFESTS) | $(NPM_EXEC) $(REPO_DEPS) scripts
 	$(NPM) install
 
 CLEAN_FILES += $(NODEUNIT) ./node_modules/nodeunit
-- 
2.21.0

